packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -268,6 +258,18 @@ class _FloatingActionButtonState extends State<FloatingActionButton> {
       child: result,
     );
 
+    if (widget.tooltip != null) {
```

Comment:
Use `new` for MergeSemantics

Reply:
I'm new to Flutter, but I know that in general the "new" keyword is [considered deprecated in dart](https://www.dartlang.org/guides/language/effective-dart/usage#dont-use-new). I realize that it still appears elsewhere in the file, I figured I would just not add it in new code and later we can clean up the existing code? WDYT?

Reply:
We're going to clean up the `new`s all at once when we're happy with the lints and fixes to remove it. In the mean time let's keep it consistent.

Reply:
SGTM, added `new`s back in.